### ISSUE TITLE
args not set on warning

### ISSUE THREAD

**Issue by BrandonHoffman** [briancurtin/deprecation#36]
```
>>> warn = deprecation.DeprecatedWarning("depricated", 1, 2)
>>> warn.args
()
```

currently warnings generated through this module do not set the args property. This is making the library incomparable with the latest release of pytest-xdist since they are using those args as a way of recreating the warning on the manager thread.

**Comment by BrandonHoffman** [comment]
see https://github.com/pytest-dev/pytest-xdist/issues/344 for more details.

**Comment by BrandonHoffman** [comment]
@briancurtin - yeah that should fix the issue

**Comment by briancurtin** [comment]
This is released in 2.0.6. Thanks for the report!
